src/allmydata/web/root.py

```diff
@@ -169,14 +169,15 @@ def renderHTTP(self, ctx):
         raise WebError("/file must be followed by a file-cap and a name",
                        http.NOT_FOUND)
 
-class IncidentReporter(RenderMixin, rend.Page):
-    def render_POST(self, ctx):
-        req = IRequest(ctx)
+class IncidentReporter(MultiFormatResource):
+    """Handler for /report_incident POST request"""
+
+    def render(self, req):
         log.msg(format="User reports incident through web page: %(details)s",
```

Comment:
In previous code, this only responded to a POST; we should probably check that here too, right?

Reply:
Ah yes, thank you for noticing!  Will update.

Reply:
Added a check for `POST` in 59f907a.